packages/verification-vue-component/src/components/Verification/item/ResultDetail.vue

```diff
@@ -266,6 +292,8 @@ export default {
                     return mdiCheckCircle
                 case 'alert':
                     return mdiAlertCircle
+                case 'alertTriangle':
+                    return mdiAlert
```

Comment:
I don't think it make sense to introduce a different shape of the alert icon for retracted. Are you shure this isn't just a layout change and the alert should be the triangle shape for all cases?

Reply:
Just discussed it with Steph, we keep the circle. I changed it back